client/src/js/modules/dc/views/gridplot.js

```diff
@@ -144,6 +148,28 @@ define(['jquery', 'marionette',
             }
         },
 
+        populateImageSelect: function() {
+            opts = []
+            for (n=1; n<=4; n++) {
```

Comment:
```suggestion
            for (let i=1; i<=4; i++) {
```

Avoid overwriting globally scoped variable `n`